hack/update-bazel-gcs-mirror.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+
+# Copyright 2018 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+KUBE_ROOT=$(dirname "${BASH_SOURCE}")/..
+
+cd "${KUBE_ROOT}"
+
+BUCKET=gs://k8s-bazel-cache
+
+gsutil acl get "${BUCKET}" > /dev/null
+
+ETCD_VERSION=$(grep -m1 ETCD_VERSION WORKSPACE | grep -o '[0-9.]*')
+
+for url in $(grep 'cache(' WORKSPACE | grep -o 'https://[^"]*'); do
```

Comment:
s/cache/mirror/?

Reply:
Woops, missed this when renaming things. Fixed!